### ISSUE TITLE
Problems of replicating

### ISSUE THREAD

**Issue by ncTimTang** [rccchoudhury/rlt#8]
Thank you for your wonderful work！I'm trying to reproduce your work, but I'm running into the following issues along the way：
1.	The test code of Kinetics-400 you provided doesn't seem to be able to run directly, I need to add “test_metadata_path” in configs/data/kinetics.yaml to run it, otherwise the dataloader will be “NoneType”. Not sure if I was right to do that.
2.	Which part of Kinetics-400 you reported in this program? Val or Test? I tried to reproduce the results, but it seemed there is a little gap. Here are my results with the Vit-B checkpoint you provided on Kinetics-400.
Val:
 
![Image](https://github.com/user-attachments/assets/4b7f9ec0-89ed-4cae-9ce1-a27f54355f77)

Test:

![Image](https://github.com/user-attachments/assets/91dee4f6-40b3-40ae-8ae0-2a329a8f0147)
 
3.	Has the checkpoint files you provided in this program already finetuned on Kinetics-400 with RLT or without RLT? Can you provide the original pretrained checkpoint. I didn’t find it in VideoMAE2 program (https://github.com/OpenGVLab/VideoMAEv2/blob/master/docs/MODEL_ZOO.md)

**Comment by VG-Huai** [comment]
Hello, have you solved this problem? I got the same result with Vit-B on test & val dataset. Turning on or off the RLT has almost no impact on the final accuracy.

**Comment by ncTimTang** [comment]
Hi @VG-Huai,  I haven't fixed this issue either. Maybe we should wait for @rccchoudhury  to answer.

**Comment by rccchoudhury** [comment]
Hi all, thanks for your interest! I'm surprised this is happening but I'll investigate and push a fix by EOD or tomorrow at the latest.

**Comment by rccchoudhury** [comment]
Hi all - I made a few fixes and pushed them. @ncTimTang thanks for the insights - there indeed was a test_metadata_path missing in kinetics.yaml. We used the **val** dataset (as do other papers evaluating on Kinetics-400) for results in the paper. I rechecked the manuscript and noticed that we did not make this completely clear, so we will update this too. Please let me know if you have any other problems!
